samples/ironside_se/secondary_boot_gen_uicr/sysbuild.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Build secondary application core image
+ExternalZephyrProject_Add(
+  APPLICATION secondary
+  SOURCE_DIR ${APP_DIR}/secondary
+  BOARD nrf54h20dk/nrf54h20/cpuapp
```

Comment:
by secondary is this just a different application in a different area, or duplicate for bootloader in a different slot? @tomchy

Reply:
It's a different application in a different area.

I wouldn't call it a bootloader as it doesn't boot anything.

I suppose the literature calls it a recovery image.

But we already had another feature called recovery image so we had to call it something else.

It's documented here,
https://github.com/nrfconnect/sdk-nrf/pull/24301/files#diff-50fe4cb5c9f65e5c7645a87bad09fd4810b5cf69d05703f267c61dd9660c39cfR441

Reply:
This should be added as a sysbuild Kconfig (and cmake file) similar to e.g. flpr or network core, see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.netcore and https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/netcore.cmake which would use that common infrastructure (and support third party boards)

Reply:
I see that there are only 5 network core samples that organized like this

Is there some kind of criteria that determines if this organization is appropriate or not?

I.e. if the image is considered to be something a customer uses in a product, or an image that just
has example code?

Reply:
Are i.e. tests considered out-of-scope for this organization?

Reply:
Hi,

I have removed these 5 lines of CMake and added the requested infrastructure in
https://github.com/nrfconnect/sdk-nrf/pull/24028/commits/b110a57c4fdefcc0cfa1be437336ab2aebddea19